kalite/distributed/static/js/distributed/topics/views.js

```diff
@@ -104,28 +115,33 @@ window.SidebarView = BaseView.extend({
     },
 
     resize_sidebar: function() {
-        if ($(window).width() < 1260)
-            this.resize_for_narrow();
+        if (this.state_model.get("open")) {
+            if ($(window).width() < 1260)
+                this.resize_for_narrow();
```

Comment:
Yeah, Travis has a linter. My immediate neighbors in the office agreed that braces were stylistically preferable even though not necessary here.